### DISCUSSION TITLE
Watermark on Draft document

### DISCUSSION THREAD

**Discussion by tsippel10** [quarto-dev/quarto-cli#4808] [Q&A]
How would I add a watermark that says "DRAFT" across each page of a Quarto document?

**Comment by mcanouil** [comment]
Depends on what is your document.

If you want it for `pdf` format then "latex watermark" in favourite search engine will give you what you need.

https://www.duckduckgo.com?q=latex+watermark

If you want it for `html` format then "HTML watermark" in your favourite search engine.

https://www.duckduckgo.com?q=html+watermark

**Comment by chris31415926535** [comment]
If you're knitting to HTML output, here are two options that you can further customize. If you put these up top after your YAML it should add a `<div>` that follows you as you scroll through the document.

This one is a simple floating header:

````
```{=html}
<div style="position: fixed; top: 0; right: 0; z-index:999;">DRAFT -- PLEASE DO NOT DISTRIBUTE</div>
```
````

This one has bolder text that floats behind the document, more like what you'll see in some Word files:

````
```{=html}
<div style="position: fixed; margin-top: 10%; margin-left:5%; font-size: xx-large; font-weight: 900; color: #CCCCCC; rotate: -45deg; z-index:-999;">DRAFT -- PLEASE DO NOT DISTRIBUTE</div>
```
````

(And for anyone wondering, to get the three backticks to render properly you need to wrap the code block in *four* backticks.)

**Comment by Steinthal** [comment]
For PDF I use [draftwatermark](https://tug.ctan.org/macros/latex/contrib/draftwatermark/draftwatermark.pdf) in my custom [eisvogel-PDF](https://github.com/Wandmalfarbe/pandoc-latex-template) extension for Quarto.

You could add

```yaml
format:
  pdf:
    header-includes: |
     \usepackage[$watermark-options$]{draftwatermark}
```


For `$watermark-options$` see the link above.
